VERSION

```diff
@@ -1 +1 @@
-0.171.12
+0.172.0
```

Comment:
I don't quite understand our versioning scheme. @igor-sirotin, are you bumping to `0.172.0` because there's a breaking change? I'm asking this because I thought when there's no breaking change we just bump the patch version.

Anyway, how we define and more importantly use versioning in status-go is hazy, but it doesn't hurt to ask :)

Reply:
oh my bad, I'll change it to `0.171.13` 👌